spec/requests/api/v1/users_spec.rb

```diff
@@ -97,6 +97,7 @@
 
         it 'should return the right error response' do
           expected_error_message = {
+            "error" => 'Validation failed',
```

Comment:
@nronas Sorry this is my bad, was just moving too quick. Should've ran specs. Will do from now on